### PR TITLE
Define the Set Timeout command

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -1117,6 +1117,61 @@
...
+    <p>
+      <table class="simple jsoncommand">
+        <tr>
+          <th>HTTP Method</th>
+          <th>Path Template</th>
+          <th>Notes</th>
+        </tr>
+        <tr>
+          <td>POST</td>
+          <td id='post-timeouts'>/session/{sessionId}/timeouts</td>
```
Comment (andreastt) [inline webdriver-spec.html]: This id attribute is not needed.

File: webdriver-spec.html [hunk 2]
```diff
@@ -1117,6 +1117,61 @@
...
+          <td></td>
+        </tr>
+      </table>
+    <p>The <dfn>Set Timeout</dfn> command sets a timeout
+    associated with the current session.</p>
```
Comment (andreastt) [inline webdriver-spec.html]: `<a>current session</a>`

File: webdriver-spec.html [hunk 3]
```diff
@@ -1117,6 +1117,61 @@
...
+    <p>The <a>remote end steps</a> for the <a>Set Timeout</a> command
+    are:</p>
+    <ol>
+      <li><p>Let <var>type</var> be the result of <a>getting a
+      property</a> named <code>type</code> from
```
Comment (andreastt) [inline webdriver-spec.html]: ´"<code>type</code>"`

Comment (andreastt) [inline webdriver-spec.html]: `"<code>type</code>"`

File: webdriver-spec.html [hunk 4]
```diff
@@ -1117,6 +1117,61 @@
...
+      the <var>parameters</var> argument.</li>
+
+      <li><p>Let <var>timeout</var> be the result of <a>getting a property</a>
+      named <code>ms</code> from the <var>parameters</var>
+      argument.from contain the type of timeout for specified
+      operations.</li>
```
Comment (andreastt) [inline webdriver-spec.html]: Clean up sentence

File: webdriver-spec.html [hunk 5]
```diff
@@ -1117,6 +1117,61 @@
...
+            to <var>timeout</var> milliseconds.</dd>
+            <dt>The string <code>"script"</code>:</dt>
+            <dd><p>Set the <a>session script timeout</a>
+            to <var>timeout</var> milliseconds.</dd>
+            <dt>Otherwise:</dt>
+            <dd><p>Return <a>error</a> with status <a>invalid
+            argument</a>.
+          </dl>
+      </li>
+      <li>Return <a>success</a> with data null</li>
```
Comment (andreastt) [inline webdriver-spec.html]: Missing full stop

**Comment by andreastt** [comment]
👍🏻 but for some fixups

**Comment by AutomatedTester** [comment]
landed in 5b057fa

**Pull request closed without merging**